screenshot/builds/master.json

```diff
@@ -17035,4 +17035,4 @@
       "isMobile": false
     }
   ]
-}
+}
```

Comment:
We keep pushing this newline back and forth. Might this be related to the cleanup script?